docs/src/main/asciidoc/_admin-guide/management-tasks/Suspend,_Resume_and_Graceful_shutdown.adoc

```diff
@@ -169,38 +169,46 @@ it will wait indefinitely.
 == Domain Mode
 
 Domain mode has similar operations as standalone mode, however they can be
-applied at both the global and server group levels:
+applied at global, host and server group levels:
 
 *Whole Domain*
 
-`:suspend-servers(timeout=x)`
+`:suspend-servers(suspend-timeout=x)`
 
 `:resume-servers`
 
-`:stop-servers(timeout=x)`
+`:stop-servers(suspend-timeout=x)`
+
+*Host level*
+
+`/host=master:suspend-servers(suspend-timeout=x)`
+
+`/host=master:resume-servers`
+
+`/host=master:shutdown(suspend-timeout=x)`
```

Comment:
A brief comment would be good after this about how the suspend in this case applies to the servers only, even though the HC itself is being shut down as well.

Perhaps put Host level after Server? That's a bit of an odd order, but it allows you to add a comment like that without having it in the middle of the simple list of ops.

Reply:
Yes, the suspend-timeout attribute could lead up in confusions in the HC shutdown operation. I moved the host section after the server section and added a note describing this situation.